src/services/kuzzleWrapper.js

```diff
@@ -121,12 +134,16 @@ export const performSearchDocuments = async (
   }
 
   const documents = result.hits.map(document => {
-    const object = {
+    let object = {
```

Comment:
should be const

Reply:
I change this object in case there is aggregation:
```
    if (result.aggregations) {
      object.aggregations = new Aggregations(result.aggregations)
    }
```

Reply:
You mutate it, but this is still the same object.

REPL example:

```
$ node
> const foo = {bar: 'baz' }
undefined
> foo.qux = true
true
> foo
{ bar: 'baz', qux: true }
> 
```

Reply:
javascript....

Reply:
For once, I disagree on that one. An object acts a bit like a hashmap: when you add a key, you mutate the container, but that doesn't change its address, so its fine.
If you want a constant object, you have to [freeze](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze) it (and all other objets/arrays in it obviously, for the same reason).

Same goes for arrays: you can have a const array and still push values in it.